From ad0d1d239afc7ad09203847793dcc020856d1035 Mon Sep 17 00:00:00 2001
From: Samuli Suominen <ssuominen@gentoo.org>
Date: Thu, 12 Jun 2014 06:30:21 -0400
Subject: [PATCH 09/10] Fix paths in libffi.pc.in

This patch was borrowed from upstream, commit
ad0d1d239afc7ad09203847793dcc020856d1035, and will be part of the next
3.1.1 release.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 configure.ac | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4f7fe3f..bbc1a9d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -590,11 +590,11 @@ AC_ARG_ENABLE(purify-safety,
 if test "x$GCC" = "xyes"; then
   if test -n "$with_cross_host" &&
      test x"$with_cross_host" != x"no"; then
-    toolexecdir='$(exec_prefix)/$(target_alias)'
-    toolexeclibdir='$(toolexecdir)/lib'
+    toolexecdir="${exec_prefix}"/'$(target_alias)'
+    toolexeclibdir="${toolexecdir}"/lib
   else
-    toolexecdir='$(libdir)/gcc-lib/$(target_alias)'
-    toolexeclibdir='$(libdir)'
+    toolexecdir="${libdir}"/gcc-lib/'$(target_alias)'
+    toolexeclibdir="${libdir}"
   fi
   multi_os_directory=`$CC $CFLAGS -print-multi-os-directory`
   case $multi_os_directory in
@@ -603,7 +603,7 @@ if test "x$GCC" = "xyes"; then
   esac
   AC_SUBST(toolexecdir)
 else
-  toolexeclibdir='$(libdir)'
+  toolexeclibdir="${libdir}"
 fi
 AC_SUBST(toolexeclibdir)
 
-- 
2.0.0

